modules/dataproc_provisioner/variables.tf

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "name" {
+  description = "Name of the profile"
+  type        = string
+}
+
+variable "label" {
+  description = "Label of the profile"
+  type        = string
+}
+
+variable "project" {
+  description = "ID of the project hosting the cluster"
+  type        = string
+  default     = "auto-detect"
+}
+
+variable "region" {
+  description = "Region of the dataproc cluster"
```

Comment:
In consistent use of "the cluster" and "the dataproc cluster".
Please standardize on "the Dataproc cluster"

Reply:
Done.